buildSrc/build.gradle.kts

```diff
@@ -10,3 +13,14 @@ repositories {
 kotlinDslPluginOptions {
     experimentalWarning.set(false)
 }
+dependencies {
+    implementation(kotlin("stdlib-jdk8"))
```

Comment:
```suggestion
    implementation(kotlin(Config.kotlinStdLib, KotlinCompilerVersion.VERSION))
```

Reply:
All changes in this file were added by Intellij and I did not mean to push them. Sorry!